### PR TITLE
adding shapesecurity to adopters

### PR DESCRIPTION
**Pull request by irivera007**
Signed-off-by: Isaac Rivera <irivera007@yahoo.com>

**What type of PR is this?**
/kind documentation

**Any specific area of the project related to this PR?**
N/A

**What this PR does / why we need it**:
Adding another adopter to the list :) <3

**Which issue(s) this PR fixes**:
N/A

**Special notes for your reviewer**:
<3

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -28,4 +28,6 @@ This is a list of production adopters of Falco (in alphabetical order):
...
 
+* [Shapesecurity/F5] (https://www.shapesecurity.com/) Shapesecurity defends against application fraud attacks like Account Take Over, Credential Stuffing, Fake Accounts, etc. Required by FedRamp certification, we needed to find a FIM solution to help monitor and protect our Kubernetes clusters. Traditional FIM solutions were not scalable and not working for our environment, but with falco we found the solution we needed. Falco's detection capabilities have helped us identify anomalous behaviour within our clusters. We leverage Sidekick (https://github.com/falcosecurity/charts/tree/master/falcosidekick) to send Falco alerts to a PubSub which in turn publishes those alerts to our SIEM (SumoLogic)
```
Comment (leodido) [inline ADOPTERS.md:31]:
```suggestion
* [Shapesecurity/F5] (https://www.shapesecurity.com/) Shapesecurity defends against application fraud attacks like Account Take Over, Credential Stuffing, Fake Accounts, etc. Required by FedRamp certification, we needed to find a FIM solution to help monitor and protect our Kubernetes clusters. Traditional FIM solutions were not scalable and not working for our environment, but with Falco, we found the solution we needed. Falco's detection capabilities have helped us identify anomalous behavior within our clusters. We leverage Sidekick (https://github.com/falcosecurity/charts/tree/master/falcosidekick) to send Falco alerts to a PubSub which in turn publishes those alerts to our SIEM (SumoLogic)
```

**Review by leodido — COMMENTED** [review]
Found a minor typo (Falco needs to be capitalized as per guidelines) 

Do you want me to pick the suggestion? Or do you want to do it yourself? :)

Thanks for sending this PR 💟

File: ADOPTERS.md (hunk 1 above)
Comment (irivera007) [reply ADOPTERS.md]: @leodido done :) thanks for flagging

**Review by leogr — APPROVED** [review]
/milestone 0.28.0

**Review by leodido — APPROVED** [review]

**Pull request merged**